package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@topsort/analytics.js",
-  "version": "1.0.3",
+  "version": "1.1.4",
```

Comment:
```suggestion
  "version": "1.0.4",
```

Reply:
should the minor version reset to 0 in that case?

Reply:
Woops, my bad

Reply:
According to https://semver.org/ 
> MINOR version when you add functionality in a backwards compatible manner, and
> PATCH version when you make backwards compatible bug fixes.

So in this case I think it's appropriate to make a patch version.